### PR TITLE
Us2.2 show wallet from api extends us2.0

### PR DESCRIPTION
**Pull request by ThiemoOrrico**
2.0 already reviewed including commit Add Wallet Service Test.

### CODE REVIEW

File: frontend/src/components/LowerButtonNavBar.js [hunk 1]
```diff
@@ -3,27 +3,14 @@ import Box from '@mui/material/Box';
...
     const ref = React.useRef(null);
-    const [messages, setMessages] = React.useState(() => refreshMessages());
-
-    React.useEffect(() => {
-        ref.current.ownerDocument.body.scrollTop = 0;
-        setMessages(refreshMessages());
-    }, [value, setMessages]);
 
     return (
         <Box sx={{ pb: 7 }} ref={ref}>
```
Comment (Flooooooooooorian) [inline frontend/src/components/LowerButtonNavBar.js:16]: Do you really need an ref here?

File: backend/src/main/java/com/tradomator/service/WalletService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+        Wallet latestWallet = null;
+        long max = 0;
+        for ( Wallet wallet : all){
+            if( wallet.getUpdateTime() > max ){
+                max = wallet.getUpdateTime();
+                latestWallet = wallet;
+            }
+        }
+        return latestWallet;
```
Comment (Flooooooooooorian) [inline backend/src/main/java/com/tradomator/service/WalletService.java:37]: Maybe create a repo function for this, should look something like this
```suggestion
    return walletRepo.findFirstByOrderByUpdatedTimeDesc()
    .orElseThrow(throw new NoSuchElementException("No wallets in DB found!"));
```

File: frontend/src/hooks/useWalletBalances.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import {useEffect, useState} from "react";
+import {getWallet} from "../service/binance-wallet-service";
+
+
+export default  function useWalletBalances(){
+    const [wallet, setWallet] = useState({})
+
+    const getWalletBalances = () =>{
```
Comment (Flooooooooooorian) [inline frontend/src/hooks/useWalletBalances.js:8]:
```suggestion
    const getWalletBalance = () =>{
```

**Review by Flooooooooooorian — APPROVED** [review]

File: backend/src/main/java/com/tradomator/model/Balance.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class Balance {
+
+    private String asset;
+    private String free;
```
Comment (christophersiem) [inline backend/src/main/java/com/tradomator/model/Balance.java:15]: maybe `unlocked` ?

File: frontend/src/components/LowerButtonNavBar.js (hunk 1 above)
Comment (ThiemoOrrico) [reply frontend/src/components/LowerButtonNavBar.js:16]: We don't need, thanks for your hint. Meanwhile changed with us3.1.

File: backend/src/main/java/com/tradomator/service/WalletService.java (hunk 2 above)
Comment (ThiemoOrrico) [reply backend/src/main/java/com/tradomator/service/WalletService.java:37]: elegant: tried to implement like this. did't work at first. With a little adaption finally worked :   
`return walletRepo.findTopByOrderByUpdateTimeDesc().orElseThrow( () -> new NoSuchElementException("Wallet Id is not set or no wallets in DB found!"));`

File: frontend/src/hooks/useWalletBalances.js (hunk 3 above)
Comment (ThiemoOrrico) [reply frontend/src/hooks/useWalletBalances.js:8]: more elegant wording , thanks

File: backend/src/main/java/com/tradomator/model/Balance.java (hunk 4 above)
Comment (ThiemoOrrico) [reply backend/src/main/java/com/tradomator/model/Balance.java:15]: point taken thanks. Will keep for now. As soon I build the mapper/ dot will adapt wording. For now i better keep like is given in external Api document.

**Pull request merged**